build/Generator/ApiGenerator.php

```diff
@@ -61,6 +61,19 @@ public function generateOperation(ServiceDefinition $definition, string $service
         $namespace->addUse($inputClass);
         $classes = $namespace->getClasses();
         $class = $classes[\array_key_first($classes)];
+        if (null !== $prefix = $definition->getEndpointPrefix()) {
+            $class->getMethod('getServiceCode')
+                ->setReturnType('string')
+                ->setVisibility(ClassType::VISIBILITY_PROTECTED)
+                ->setBody('return ' . \var_export($prefix, true) . ';');
```

Comment:
Why `var_export`?  This will always be a string, right?

Btw, This is the first time I see `var_export` =)

Reply:
this is a very safe way to escape a variable.

symfony uses var_export a lot when it generates code.